content/en/docs/tutorials/hello-minikube.md

```diff
@@ -136,6 +136,8 @@ Kubernetes [*Service*](/docs/concepts/services-networking/service/).
 
     The `--type=LoadBalancer` flag indicates that you want to expose your Service
     outside of the cluster.
+    
+    Please note that the Pod won't show up under ports other than 8080.
```

Comment:
This note doesn't explain what thing has that port 8080 restriction. Is it Minikube? Kubernetes?
The reader will want to know.

How about:
```suggestion
    The application code inside the image `k8s.gcr.io/echoserver` only listens on TCP port 8080. If you used
    `kubectl expose` to expose a different port, clients could not connect to that other port.
```

Reply:
I definitely agree with you. It should be explained that the **app itself** inside container listens on `8080`. Thanks!